components/colorarea/stories/template.js

```diff
@@ -0,0 +1,36 @@
+// template is where the component markup with go
+
+import { html } from 'lit-html';
+import { classMap } from 'lit-html/directives/class-map.js';
+import { ifDefined } from 'lit-html/directives/if-defined.js';
+import { Template as ColorHandle } from "@spectrum-css/colorhandle/stories/template.js";
+
+import '../index.css';
+
+export const Template = ({
+  rootClass = "spectrum-ColorArea",
+  customClasses = [],
+  isDisabled = false,
+  customSize,
+  handlePosition,
+  style = "--spectrum-picked-color: rgb(255, 0, 0);" + handlePosition,
+  // ...globals
+}) => {
+  return html`
+    <div class=${classMap({
+    [rootClass]: true,
+    'is-disabled': isDisabled,
+    ...customClasses.reduce((a, c) => ({ ...a, [c]: true }), {}),
+  })} ?disabled=${isDisabled}} style=${ifDefined(customSize)}>
+  <div class="spectrum-ColorArea-gradient" style="background: linear-gradient(to top, black 0%, rgba(0, 0, 0, 0) 100%), linear-gradient(to right, white 0%, rgba(0, 0, 0, 0) 100%), rgb(255, 0, 0);"></div>
+  ${ ColorHandle({
```

Comment:
Awesome! Don't forget to pass globals down to the nested component's template so it is aware of theme contexts.

Reply:
Got this added to the template for ColorArea.
To make sure I am understanding, do we only use this on nested components?

Reply:
Yes exactly - it's just ensuring that express theme, light/dark theme, right-to-left, etc. are passed to nested templates. They don't seem to inherit them on their own.